src/search.js

```diff
@@ -79,16 +83,16 @@ module.exports = function(list) {
     list.trigger('searchStart');
 
     prepare.resetList();
-    prepare.setSearchString(str);
+    prepare.setSearchPattern(str);
     prepare.setOptions(arguments); // str, cols|searchFunction, searchFunction
     prepare.setColumns();
 
-    if (searchString === "" ) {
+    if (!searchPattern ) {
       search.reset();
     } else {
       list.searched = true;
       if (customSearch) {
-        customSearch(searchString, columns);
+        customSearch(searchPattern, columns);
```

Comment:
This might be a breaking change, depending on how people use the customSearch option.

Reply:
`searchPattern.source` could help.